pkg/user/user_integration_test.go

```diff
@@ -573,7 +531,7 @@ func TestUserHandler(t *testing.T) {
 			var users []model.User
 			client.GetJSON(t, "/users", &users, inttest.WithAuthToken(adminToken.AccessToken))
 
-			expectedNumberOfUsers := int(userCounter.Load())
+			expectedNumberOfUsers := int(userCounter.Load()) + 3 // counter + 1 admin + 1 manually created + 1 manually created which isn't validated
```

Comment:
you could increment the counter at the sites where these users are create

Reply:
True but not for the admin, I've updated the code